src/platform/windows/win_snprintf.c

```diff
@@ -0,0 +1,34 @@
+#ifndef NNG_WIN_SNPRINTF_H
+#define NNG_WIN_SNPRINTF_H
+
+#if !NNG_HAVE_SNPRINTF
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+
+#include <stdarg.h> // NOLINT
+#include <stdio.h>  // NOLINT
+
+#if !defined(_TRUNCATE)
```

Comment:
This is truly disgusting.  If the tool chain *also* doesn't support _TRUNCATE we must be *really* old?

Reply:
Also, is the value of that symbol documented this way?

Reply:
Oh, I see, this is probably MinGW-32... No thanks.  Explicitly we don't support mingw-32, and never will.  Mingw-64 is fine.  (mingw-32 lacks some APIs we depend on, and it would be non-trivial to fix those gaps without leaving unacceptable race conditions in the code.)

Reply:
I will remove this.